src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/BaseTest.java

```diff
@@ -42,4 +61,31 @@ private Jwt dummyJwt() {
                 .build();
     }
 
+    protected HttpHeaders createHttpHeaders(String serviceName) {
+        HttpHeaders headers = new HttpHeaders();
+        headers.add(AUTHORIZATION, "Bearer EncodedAuthToken");
+        String s2SToken = generateDummyS2SToken(serviceName);
+        headers.add(SERVICE_AUTHORIZATION, s2SToken);
```

Comment:
please add `headers.add(AuditConfiguration.REQUEST_ID, EXAMPLE_REQUEST_ID);` like CPO which is useful for audit log checks